config/application.rb

```diff
@@ -29,5 +29,7 @@ class Application < Rails::Application
     config.time_zone = 'London'
 
     config.active_record.raise_in_transactional_callbacks = true
+    config.unsubscribe_url =
+      '<https://www.prisonvisits.service.gov.uk/unsubscribe>'
```

Comment:
This needs an accompanying endpoint. In the old app, it's just a file in `public/`.